beaker.gemspec

```diff
@@ -38,7 +38,7 @@ Gem::Specification.new do |s|
   ## aws-sdk-2 doesn't require json, so we can give this up when we move
 
   s.add_runtime_dependency 'hocon', '~> 1.0'
-  s.add_runtime_dependency 'net-ssh', '~> 3.2'
+  s.add_runtime_dependency 'net-ssh', '3.3.0beta1'
```

Comment:
looks like this needs to be `3.3.0.beta1`? Gem wasn't on our mirror (yay manual mirroring), but it failed to mirror with the version string here.